benchmark_runner/common/oc/oc.py

```diff
@@ -529,8 +529,8 @@ def delete_available_released_pv(self):
             if pv_status == 'Available' or pv_status == 'Released':
                 available_pv = self.run(fr"{self.__cli} get pv -o jsonpath={{.items[{ind}].metadata.name}}")
                 logger.info(f'Delete {pv_status} pv {available_pv}')
-                self.run(fr"{self.__cli} delete localvolume -n openshift-local-storage local-disks")
-                self.run(fr"{self.__cli} delete pv {available_pv}")
+                self.run(fr"{self.__cli} delete localvolume -n openshift-local-storage local-disks --wait=false")
+                self.run(fr"{self.__cli} delete pv {available_pv} --wait=false")
```

Comment:
Why do you want to wait?

Reply:
@RobertKrawitz, currently, all LSO workloads have failed due to this issue in ODF 4.17
We need to merge it ASAP to get results for CPT 4.17.